### PR TITLE
Ld30

### PR DESCRIPTION
**Pull request by bohdan-rebrov**

### CODE REVIEW

File: Server/app.js [hunk 1]
```diff
@@ -39,15 +39,15 @@ auth(app);
 app.use('/', routes);
 app.use('/', menu);
 app.use('/categories', categories);
-app.use(function (req, res, next) {
+/*app.use(function (req, res, next) {
```
Comment (chapluck) [inline Server/app.js]: Remove comment symbols /*

File: Server/models/user.model.js [hunk 2]
```diff
@@ -11,7 +11,9 @@ module.exports = function (sequelize, DataTypes) {
 
 		email:{ type: DataTypes.STRING, 
 			allowNull: false,			 
-			 validate: {len: [2,50], isEmail: true}			},
+			 validate: {len: [2,50], isEmail: true},
+
```
Comment (chapluck) [inline Server/models/user.model.js]: remove redundant empty line

File: Server/routes/users.route.js [hunk 3]
```diff
@@ -1,17 +1,55 @@
...
+	userProvider.getById(req.params.userId)
+	.then(result => res.json(result))
+	.catch(next);
 });
 
 router.use('/:userId/orders', require('./orders.route'));
 
-module.exports = router;
\ No newline at end of file
+module.exports = router;
```
Comment (chapluck) [inline Server/routes/users.route.js]: move exports assignment to the end of the file

File: Server/services/user.provider.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+	},
+
+	getById: function (userId) {
+		return db.User.findById(userId).then(user =>
+			user || Promise.reject(new errors.NotFound("User is not found")));
+	},
+
+	add: function (user) {
+		return db.User.create(user).catch(user =>
+			Promise.reject(new errors.BadRequest("Validation error (Please check you input)"))
```
Comment (chapluck) [inline Server/services/user.provider.js]: add UniqueConstraintError handling

**Review by chapluck — CHANGES_REQUESTED** [review]

**Comment by bohdan-rebrov** [comment]
It is done

**Review by chapluck — APPROVED** [review]

**Pull request merged**